common/djangoapps/util/tests/test_db.py

```diff
@@ -222,6 +222,7 @@ class MigrationTests(TestCase):
     """
     Tests for migrations.
     """
+    @unittest.skip("Migration will delete several models. Need to ship not referencing it first. DEPR-12.")
```

Comment:
@jmbowman FYI, I had to add this skip right here due to the microsites migrations.

Reply:
Ok, just remember to turn this back on afterwards.